.github/workflows/deploy.yml

```
@@ -30,5 +30,18 @@ jobs:
       - name: 🐳 Build Docker image
         run: docker build -t miproyecto-base .
 
+      - name: 🐳 Log in to Docker Hub
+        uses: docker/login-action@v2
```

actionlint (1.7.7)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)